ts/state/selectors/conversations.ts

```diff
@@ -880,171 +881,147 @@ export const getMessagePropsByMessageId = createSelector(
 export const getMessageAvatarProps = createSelector(getMessagePropsByMessageId, (props):
   | MessageAvatarSelectorProps
   | undefined => {
-  if (!props || _.isEmpty(props)) {
+  if (!props || isEmpty(props)) {
     return undefined;
   }
 
-  const {
-    authorAvatarPath,
-    authorName,
-    sender,
-    authorProfileName,
-    conversationType,
-    direction,
-    isPublic,
-    isSenderAdmin,
-  } = props.propsForMessage;
-
-  const { lastMessageOfSeries } = props;
-
   const messageAvatarProps: MessageAvatarSelectorProps = {
-    authorAvatarPath,
-    authorName,
-    sender,
-    authorProfileName,
-    conversationType,
-    direction,
-    isPublic,
-    isSenderAdmin,
-    lastMessageOfSeries,
+    lastMessageOfSeries: props.lastMessageOfSeries,
+    ...pick(props.propsForMessage, [
+      'authorAvatarPath',
+      'authorName',
+      'sender',
+      'authorProfileName',
+      'conversationType',
+      'direction',
+      'isPublic',
+      'isSenderAdmin',
+    ]),
   };
 
   return messageAvatarProps;
 });
 
+export const getMessageReactsProps = createSelector(getMessagePropsByMessageId, (props):
+  | MessageReactsSelectorProps
+  | undefined => {
+  if (!props || isEmpty(props)) {
+    return undefined;
+  }
+
+  const msgProps: MessageReactsSelectorProps = pick(props.propsForMessage, [
+    'convoId',
+    'conversationType',
+    'isPublic',
+    'reacts',
+    'serverId',
+  ]);
+
+  if (msgProps.reacts) {
+    const sortedReacts = Object.entries(msgProps.reacts).sort((a, b) => {
+      return a[1].index < b[1].index ? -1 : a[1].index > b[1].index ? 1 : 0;
+    });
```

Comment:
because those are just numbers, you can do this I think
`return b[1].index - a[1].index`

Reply:
Unfortunately this doesn't work